config/generate/template.py

```diff
@@ -0,0 +1,25 @@
+import os
+import binascii
+
+def render_template(template, destination, **kwargs):
+    with open(template, 'r') as f:
+        raw_template = f.read()
+        rendered = raw_template.format(**kwargs)
+
+        with open(destination, 'w+') as output:
+            output.write(rendered)
+
+backend = raw_input("Which backend to use ? (1 - in-memory, 2 - postgresql, 3 - redis) :")
+if backend == '':
+    backend = 2;     #default
+
+if backend == 2:
```

Comment:
Of course, it's missing the other options here :)

I would build a dictionary depending the answers that are given to the answers. Eg. start with a clean one `options={}`
And then, depending the answer to the questions, add the correct values in there.